### ISSUE TITLE
有测试过albert-tiny的decode，encode的例子吗？

### ISSUE THREAD

**Issue by kaixinbuyu** [natureLanguageQing/bert4keras#1]
因为各种原因只能装tf1.12-keras2.2.4导致无法用最新的bert4keras。
后来测试了这个版本能加载albert。
但是测试了下例子程序，bert正常，albert直接decode回来的句子竟然和输入不一样，多了几个##？

#! -*- coding: utf-8 -*-
# 测试代码可用性
#
from bert4keras.bert import load_pretrained_model
from bert4keras.utils import SimpleTokenizer, load_vocab
import numpy as np

#/home/go/workspace/document/dataset_test/albert_tiny
# from bert4keras.bert import load_pretrained_model
# from bert4keras.utils import SimpleTokenizer,load_vocab

config_path = '/home/go/workspace/document/dataset_test/albert_tiny/albert_config_tiny.json'
checkpoint_path = '/home/go/workspace/document/dataset_test/albert_tiny/albert_model.ckpt'
dict_path = '/home/go/workspace/document/dataset_test/albert_tiny/vocab.txt'

# config_path = '/home/go/workspace/document/dataset_test/albert_tiny_google/albert_config_tiny_g.json'
# checkpoint_path = '/home/go/workspace/document/dataset_test/albert_tiny_google/albert_model.ckpt'
# dict_path = '/home/go/workspace/document/dataset_test/albert_tiny_google/vocab.txt'

token_dict = load_vocab(dict_path) # 读取词典
tokenizer = SimpleTokenizer(token_dict) # 建立分词器

#tokenizer = Tokenizer(dict_path)

#model = build_bert_model(config_path, checkpoint_path,albert=True)#, seq_len=None
model = load_pretrained_model(config_path, checkpoint_path,albert=True, with_mlm=True) # 建立模型，加载权重 是否包含MLM部分

# 编码测试
# token_ids, segment_ids = tokenizer.encode(u'语言模型')
# print(model.predict([np.array([token_ids]), np.array([segment_ids])]))

# mask
token_ids, segment_ids = tokenizer.encode(u'科学技术是第一生产力')
print('token_ids',token_ids)
print('segment_ids',segment_ids)

# mask掉“技术”
token_ids[3] = token_ids[4] = token_dict['[MASK]']
# 用mlm模型预测被mask掉的部分
probas = model.predict([np.array([token_ids]), np.array([segment_ids])])[0]
print('probas',probas)

print('decode-probas',tokenizer.decode(probas.argmax(axis=1))) # 结果
'''
的科学技术是第##一生##产。。
'''
